dashboard/rules.py

```diff
@@ -38,7 +39,7 @@ def is_instructor_in_course(user, course):
 def is_above_student_in_course(user, course):
     try:
         enrolled = User.objects.get(
-            sis_name=user.get_username(),
+            sis_name=Q(user.get_username()) | Q(sis_id=user.get_username()),
```

Comment:
use "Q(sis_name=user.get_username()) | Q(sis_id=user.get_username())" instead?

In fact, there are multiple occurrence of "Q(sis_name=user.get_username()) | Q(sis_id=user.get_username())" form, maybe that could be generalized.

Reply:
Yeah this is a good idea, I think we can put these in the model and clean this up.

Reply:
I moved these into the model and removed an unused method